artifacts/camara-cloudevents/event-subscription-template.yaml

```diff
@@ -0,0 +1,1513 @@
+openapi: 3.0.3
+info:
+  title: Notification Subscription Template
+  description: |
+    This file is a template for CAMARA API explicit subscription endpoint and for Notification model. Additional information are provided in API Design Guidelines document.
+
+  termsOfService: http://swagger.io/terms/
+  contact:
+    email: project-email@sample.com
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+  version: 0.1.0-wip
+externalDocs:
+  description: Product documentation at CAMARA
+  url: hhttps://github.com/camaraproject/
+servers:
+  - url: '{apiRoot}/event-type-notification-subscriptions/v0'
+    variables:
+      apiRoot:
+        default: http://localhost:9091
+        description: API root
+tags:
+  - name: event-type notification subscription
+    description: Operation to manage event subscription on event-type event 
+
+paths:
+  /subscriptions:
+    post:
+      tags:
+        - event-type notification subscription
+      summary: 'Create a event-type event subscription'
+      description: Create a event-type event subscription
+      operationId: createEvent-TypeSubscription
+      parameters:
+        - $ref: '#/components/parameters/x-correlator'
+      security:
+        - openId:
+          - api-name:subscriptions:create
```

Comment:
@bigludo7 Does `api-name:subscriptions:create` represent the wildcard scope?

Reply:
No. This scope is for checking the right to create a new subscription.
As we have for this API scope to check operation & scope to capture consent I'm not sure about the relevance of having a Wild card scope.

Reply:
ok understood. BTW, isn't "subscription" part of the API name?

Reply:
Not in all case but usually yes. The exception is geofencing.
As the scope definition is `api-name:[resource:]action` we are fine.
We can have
`geofencing-subscriptions-create`  (here with the resource)
`sim-swap-notification-subscription-create` (here without the resource)

WDYT?